pkg/apis/app/v1alpha1/kogitobuild_types.go

```diff
@@ -19,6 +19,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// LabelKeyAppName is the default label added to all resources
+	LabelKeyAppName = "app"
```

Comment:
We have lots of this constant in the code, I believe the one we use the most is in the `infrastructure` package. Could we keep it in one place?

Reply:
done